Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -128,11 +128,18 @@ function New-WhiskeySemanticVersion
                 return
             }
 
-            $csprojVersion = $csprojXml.SelectNodes('/Project/PropertyGroup/Version') | Select-Object -ExpandProperty '#text'
+            $xmlns = New-Object System.Xml.XmlNamespaceManager($csprojXml.NameTable)
+            $xmlns.AddNamespace('ns', $csprojXml.DocumentElement.NamespaceURI)

```

Comment:
What happens if there isn't a namespace? If the `xmlns` attribute on the project is missing? Add a test for that and see what happens.

Reply:
The existing test immediately above this one already handles this condition.